internal/operator/apply.go

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier:Apache-2.0
```

Comment:
same as above, plus if there's a strong reason for not having a /operator folder, let's give this a bit of structure with internal/operator/helm , internal/operator/controller, etc